### PR TITLE
WEBUMENIA-1467 Automatically deploy "develop" branch to test

### PR DESCRIPTION
**Pull request by eronisko**
# Description

Successful run: https://github.com/SlovakNationalGallery/webumenia.sk/actions/runs/336393101

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by eronisko** [comment]
Ah, looks like this broke some tests, hang on...

**Comment by eronisko** [comment]
Ready now. I've gone down a rabbit hole of `composer update` and had to come up the way I had come :D

File: .github/workflows/develop-autodeploy.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    - name: Restore Composer cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-composer-
+    - uses: shivammathur/setup-php@v2
+      with:
+        php-version: 7.4
```
Comment (igor-kamil) [inline .github/workflows/develop-autodeploy.yml:26]: for test.webumenia.sk we use `php7.2` for now. 
(locally I already use php7.4 for webumenia development and it works nicely... so it shouldn't be any issue)
maybe we can update also virtual host to start using `php7.4-fpm` later

**Review by igor-kamil — APPROVED** [review]
LGTM

File: config/elasticsearch/client.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 return [
     'hosts' => [
```
Comment (rastislav-chynoransky) [inline config/elasticsearch/client.php:4]: Doesn't this need to be an array of hosts?

Comment (eronisko) [reply config/elasticsearch/client.php:4]: It still is: https://github.com/SlovakNationalGallery/webumenia.sk/blob/e6e6f0f4059f636053c3de7c6c1594e408e5a833/config/elasticsearch/client.php

Comment (eronisko) [reply config/elasticsearch/client.php:4]: Now I get what you meant :) Well spotted.

File: config/elasticsearch/client.php [hunk 3]
```diff
@@ -2,6 +2,7 @@
...
-        sprintf('http://%s:%d', env('ES_HOST'), env('ES_PORT'))
+        'host' => env('ES_HOST', 'localhost'),
+        'port' => env('ES_PORT', '9200'),
```
Comment (rastislav-chynoransky) [inline config/elasticsearch/client.php:6]:
```suggestion
        [
                'host' => env('ES_HOST', 'localhost'),
                'port' => env('ES_PORT', '9200'),
        ]
```
I mean like this. Not sure if that would not be interpreted as two hosts: `localhost`, `9200`.

Comment (eronisko) [reply config/elasticsearch/client.php:6]: You're totally right. This PR is already merged in, but here's the fix: https://github.com/SlovakNationalGallery/webumenia.sk/pull/380

**Pull request merged**